french/manger/trou/reveille/surprise.md

```diff
@@ -0,0 +1,11 @@
+Et la surprise ! Tout cela n'était pas un rêve.
+
+En effet, vous vous réveillez bien au milieu d'un plan d'eau, la peau toujours recouverte d'une fine pellicule caramélisée.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.